### DISCUSSION TITLE
Inquiry About Serenity Support for .NET Core 5 and 6

### DISCUSSION THREAD

**Discussion by EdwinThomas11** [serenity-is/Serenity#7214] [Help] [answered]
Hi,

I need some quick information regarding supported versions.

I am currently using .NET Core 3.1 and Serenity 3.12.6. I need to upgrade to .NET Core 5 or 6. Does Serenity support .NET 5 or 6?

Regards,  
Edwin Thomas D

**Comment by ra-bedo11** [comment]
.net 6 or 8

**Comment by VictorTomaili** [comment, marked as answer]
https://github.com/serenity-is/Serenity/issues/6337

https://github.com/serenity-is/Serenity/issues/7020
